app/src/main/java/com/example/asiantech/playmusic/fragment/PlaySongFragment.java

```diff
@@ -0,0 +1,251 @@
+package com.example.asiantech.playmusic.fragment;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.os.Handler;
+import android.support.v4.app.Fragment;
+import android.support.v4.view.ViewPager;
+import android.util.Log;
+import android.widget.ImageView;
+import android.widget.SeekBar;
+import android.widget.TextView;
+
+import com.example.asiantech.playmusic.MainActivity;
+import com.example.asiantech.playmusic.R;
+import com.example.asiantech.playmusic.adapter.PlaySongAdapter;
+import com.example.asiantech.playmusic.service.MusicService;
+
+import org.androidannotations.annotations.AfterViews;
+import org.androidannotations.annotations.Click;
+import org.androidannotations.annotations.EFragment;
+import org.androidannotations.annotations.ViewById;
+
+import java.util.Formatter;
+import java.util.Locale;
+
+import me.relex.circleindicator.CircleIndicator;
+
+/**
+ * @author HoaHT
+ */
+@EFragment(R.layout.play_song_fragment)
+public class PlaySongFragment extends Fragment {
+    @ViewById(R.id.tvSongTitle)
+    TextView mTvSongTitle;
+    @ViewById(R.id.seekBar)
+    SeekBar mSeekBar;
+    @ViewById(R.id.tvCurrentTime)
+    TextView mTvCurrentTime;
+    @ViewById(R.id.tvSongTime)
+    TextView mTvSongTime;
+    @ViewById(R.id.imgPlay)
+    ImageView mImgPlay;
+    @ViewById(R.id.imgMode)
+    ImageView mImgMode;
+    @ViewById(R.id.pager)
+    ViewPager mViewPager;
+    @ViewById(R.id.indicator)
+    CircleIndicator mIndicator;
+    private MusicService mMusicService;
+    private StringBuilder mFormatBuilder;
+    private Formatter mFormatter;
+    private Handler mHandler = new Handler();
+    private int mMode;
+
+    @AfterViews
+    void afterView() {
+        Log.d("TAG PLAY_SONG", "afterView");
```

Comment:
WARNING: [AvoidDuplicateLiterals] The String literal "TAG PLAY_SONG" appears 7 times in this file; the first occurrence is on line 59
http://pmd.sourceforge.net/pmd-5.2.3/pmd-java/rules/java/strings.html#AvoidDuplicateLiterals